src/main/java/net/fabricmc/installer/util/CrashDialog.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.installer.util;
+
+import javax.swing.*;
+import java.awt.*;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public class CrashDialog {
+
+	public CrashDialog(Throwable throwable) {
+		JFrame frame = new JFrame("Fabric installer has crashed!");
```

Comment:
consider translating this with the [util bundle](https://github.com/FabricMC/fabric-installer/blob/cfe5a39132d5ff4ceb4b3421229f5c54bbc18134/src/main/java/net/fabricmc/installer/util/Utils.java#L36)?

Reply:
@modmuss50

Reply:
No, I think keeping this basic crash dialong free of as much code behind it is best. Idealy people will never see it